ddtrace/contrib/django/patch.py

```diff
@@ -30,8 +30,19 @@ def traced_setup(wrapped, instance, args, kwargs):
             else:
                 settings.MIDDLEWARE_CLASSES.insert(0, 'ddtrace.contrib.django.TraceMiddleware')
 
+        if 'ddtrace.contrib.django.TraceExceptionMiddleware' not in settings.MIDDLEWARE_CLASSES:
+            if isinstance(settings.MIDDLEWARE_CLASSES, tuple):
+                # MIDDLEWARE_CLASSES is a tuple < 1.9
+                settings.MIDDLEWARE_CLASSES = settings.MIDDLEWARE_CLASSES + \
+                        ('ddtrace.contrib.django.TraceExceptionMiddleware')
```

Comment:
should we still add `TraceExceptionMiddleware` here? isn't it handled by the `insert_exception_middleware`?